### ISSUE TITLE
use puppet code instead of ruby code in template

### ISSUE THREAD

**Issue by Phil-Friderici** [sensu/sensu-puppet#731]
sensu-version.erb is only used to manipulate a string.
This should be rather done in puppet code instead.

**Comment by Phil-Friderici** [comment]
Same is true for:

```
sensu-puppet$ grep -ri scope.lookupvar .
./manifests/extension.pp:  $filename = inline_template('<%= scope.lookupvar(\'source\').split(\'/\').last %>')
./manifests/handler.pp:    $filename = inline_template('<%= scope.lookupvar(\'source\').split(\'/\').last %>')
./manifests/mutator.pp:    $filename = inline_template('<%= scope.lookupvar(\'source\').split(\'/\').last %>')
./manifests/plugin.pp:      $filename = inline_template('<%= scope.lookupvar(\'name\').split(\'/\').last %>')
./manifests/plugin.pp:      $filename = inline_template('<%= scope.lookupvar(\'name\').split(\'/\').last %>')
```

**Comment by ghoneycutt** [comment]
Fix released in v2.22.0
